json-endpoints/jetpack/class.jetpack-json-api-protect-whitelist.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+class Jetpack_JSON_API_Protect_Whitelist extends Jetpack_JSON_API_Endpoint {
+	protected $needed_capabilities = 'activate_plugins';
+
+	public function callback( $path = '', $blog_id = 0, $object = null ) {
```

Comment:
I wouldn't redefine the callback method here as it's doing almost the same as the one defined in the parent class.